schemas/context/webinfo.schema.json

```diff
@@ -30,6 +30,14 @@
           "$ref": "https://ns.adobe.com/xdm/context/webreferrer",
           "description":
             "The referrer of a web interaction, which is the URL a visitor came from immediately before the current web interaction was recorded."
+        },
+        "https://ns.adobe.com/xdm/data/metrics/web/link-clicks": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Click of a web-link has occurred."
+        },
+        "https://ns.adobe.com/xdm/data/metrics/web/page-views": {
```

Comment:
I would put this under `xdm:webPageView`

Reply:
I am concerned about littering metrics in multiple levels of the hierarchy if we can get around it. I would love more peoples opinions on this.

Reply:
The benefit is putting the metric right next to the dimension you are most likely looking for to break it down.

Reply:
@cdegroot-adobe, Can we move this under `xdm:webPageView`?

Reply:
done